server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -255,6 +256,7 @@ export class DeliLambda extends TypedEventEmitter<IDeliLambdaEvents> implements
     }
 
     public handler(rawMessage: IQueuedMessage) {
+        this.emit("close", LambdaCloseType.Stop);
```

Comment:
Why are we emitting close here? Should it not be emitted in close()?

Reply:
Oops, mistakenly committed the wrong thing, fixed